FormalConjectures/ErdosProblems/68.lean

```diff
@@ -0,0 +1,33 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Erdős Problem 68
+
+*Reference:* [erdosproblems.com/68](https://www.erdosproblems.com/68)
+-/
+
+/--
+Is
+$$\sum_{n=2}^\infty \frac{1}{n!-1}$$
+irrational?
+-/
+@[category research open, AMS 11]
+theorem erdos_68 :
+    Irrational (∑' n : ℕ, 1 / ((n+2).factorial-1 : ℝ)) ↔ answer(sorry) := by
+  sorry
```

Comment:
Would you mind also adding the alternate formula for this sum from https://www.erdosproblems.com/68?